### PR TITLE
Update related displays

### LINKED ISSUE #511: Update related display UX to new architecture.

### PR DESCRIPTION
**Pull request by dpenciso**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -207,14 +207,12 @@ window.trelliscopeApp = trelliscopeApp;
...
-// trelliscopeApp('960f51e6', '_test/trelliscope-examples3/gapminder_bells/config.jsonp', { logger: true });
-trelliscopeApp('44c922eb', '_test/trelliscope-examples3/gapminder_reldisp/config.jsonp', { logger: true });
-// trelliscopeApp('1060b383', '_test/trelliscope-examples3/network_nonraster/config.jsonp', { logger: true });
-// trelliscopeApp('0d3d4590', '_test/trelliscope-examples3/pokemon/config.jsonp', { logger: true });
+// trelliscopeApp('097af825', '_test/trelliscope-examples3/gapminder/config.jsonp', { logger: true });
 // trelliscopeApp('80222985', '_test/gapminder_coggroups/config.json', { logger: true });
 // trelliscopeApp('96c61ca5', '_test/trelliscope-examples2/gapminder_reldisp/config.jsonp', { logger: true });
 // trelliscopeApp('17a6ca23', '_test/trelliscope-examples2/network_nonraster/config.jsonp', { logger: true });
 // trelliscopeApp('96c61ca5', '/config.json', { logger: true, mockData: true });
+trelliscopeApp('44c922eb', '_test/trelliscope-examples3/gapminder_reldisp/config.jsonp', { logger: true });
```
Comment (Incin) [inline src/index.tsx:215]: probably wouldnt commit this.

**Review by Incin — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 1 above)
Comment (dpenciso) [reply src/index.tsx:215]: I thought I got the ok from Jeff to do it on this branch since it's our test branch?

**Review by jefferymills — APPROVED** [review]
👶

File: src/index.tsx (hunk 1 above)
Comment (jefferymills) [reply src/index.tsx:215]: Either way I need to merge it so I can work on the Panel component.

**Pull request merged**